src/components/GaugeWeight.tsx

```diff
@@ -27,7 +27,17 @@ export default function GaugeWeight({
   const chartComponentRef = useRef<HighchartsReact.RefObject>(null)
   const basicPools = useContext(BasicPoolsContext)
   const theme = useTheme()
-  if (basicPools == undefined) return <CircularProgress color="secondary" />
+  if (basicPools == undefined)
```

Comment:
I'm sure you didn't add this but can you change this to check for all falsy values instead of explicitly checking `undefined`? 

```suggestion
  if (!basicPools)
```